frontend/src/components/VMediaInfo/VMediaTags.vue

```diff
@@ -1,25 +1,45 @@
 <template>
   <ul v-if="tags.length" class="flex flex-wrap gap-2">
+    <div v-if="additionalSearchViews">
+      <VTag
+        v-for="(tag, index) in tags"
+        :key="index"
+        href="/"
+        :title="tag.name"
+      />
+    </div>
     <VMediaTag v-for="(tag, index) in tags" :key="index" tag="li">{{
```

Comment:
If you add `v-else` to `VMediaTag` here, it will be hidden when `additionalSearchViews` is true, so there'll be no duplication of tags.